tutorials-dp0-3/major-updates-log-dp0-3.rst

```diff
@@ -50,6 +51,21 @@ and clicking on "history" (near upper-right).
 Major Updates Log
 =================
 
+Mar 6 2025
+----------
+
+All Jupyter Notebook tutorials were migrated to the new repository `lsst/tutorial-notebooks <https://github.com/lsst/tutorial-notebooks>`_.
+This repository no longer has a ``/data`` folder.
+Instead, input data for notebook tutorials is stored in the new Git LFS repository `lsst/tutorial-notebooks-data <https://github.com/lsst/tutorial-notebooks-data>`_.
```

Comment:
Same comment as on dp02 site -- should we give the path to the data/ folder at RSP?

Reply:
made same change